### ISSUE TITLE
Compiling on Linux with Python 3 raises error

### ISSUE THREAD

**Issue by snorfalorpagus** [bradfordboyle/pyglpk#6]
Compiling on Linux (OpenSUSE 13.2) against Python 3 raises the following error:

```
snorf@linux-4qyv:~/Desktop/pyglpk> python3 setup.py build_ext --inplace
running build_ext
building 'glpk' extension
creating build
creating build/temp.linux-x86_64-3.4
creating build/temp.linux-x86_64-3.4/src
gcc -pthread -Wno-unused-result -Werror=declaration-after-statement -DNDEBUG -fmessage-length=0 -grecord-gcc-switches -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector -funwind-tables -fasynchronous-unwind-tables -g -DOPENSSL_LOAD_CONF -fPIC -I/usr/include -I/usr/include/python3.4m -c src/glpk.c -o build/temp.linux-x86_64-3.4/src/glpk.o
src/glpk.c: In function ‘PyInit_glpk’:
src/glpk.c:76:3: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
   char myversion[10];
   ^
cc1: some warnings being treated as errors
error: command 'gcc' failed with exit status 1
```

A quick search reveals an answer: http://stackoverflow.com/questions/13291353/iso-c90-forbids-mixed-declarations-and-code-in-c

```
CFLAGS="-std=c99" python3 setup.py build_ext --inplace
```

Is this just a strange default on my system? Do we address it in pyglpk (in `setup.py` or in the documentation)?

**Comment by bradfordboyle** [comment]
Which version of GCC are you using? On stock Debian 8.0, I get a clean compile. I can force a failure by setting `CFLAGS="-std=c90"` before building, but I get errors about C++ style comments.

**Comment by snorfalorpagus** [comment]
I'm just using whatever is the default for OpenSUSE 13.2:

```
snorf@linux-4qyv:~/Desktop/pyglpk> gcc --version
gcc (SUSE Linux) 4.8.3 20140627 [gcc-4_8-branch revision 212064]
```

**Comment by bradfordboyle** [comment]
I just spun up an openSUSE 13.2 VM and got the same error. I have not been able to reproduce this error message with any of the options for `-std`.

We can correct this in `setup.py` with the option `extra_compile_args` for the `Extension` object. I would suggest we use `-std=gnu90` as this is supposed to be the default for [gcc-4.8.x](https://gcc.gnu.org/onlinedocs/gcc-4.8.3/gcc/Standards.html).

**Comment by bradfordboyle** [comment]
Fix pushed to d6bff1a.
